content/blog/2016/2016-04-14-jenkins-world-registration-is-open.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Registration is Open for Jenkins World 2016!"
+tags:
+- jenkins world
```

Comment:
Please remove the space, either `jenkinsworld` (my preference) or `jenkins-world`